sample-apps/javascript-sample-app/README.md

```diff
@@ -0,0 +1,21 @@
+## Javascript Opentelemetry Sample App
+
+### Description
+This is a Javascript auto-instrumented sample app demonstrating the features of OTel. Tracing is automatically instrumented, but there are also some aspects of manual instrumentation involved as well.
+
+### Getting Started:
```

Comment:
The README is pretty lacking on what is actually exposed and what the sample app does. Can we use [other sample apps ](https://github.com/aws-observability/aws-otel-community/tree/master/sample-apps/ruby-rails-sample-app)as reference?

Reply:
you can just copy paste the structure of the java and go sample apps and make the adjustments.